README.md

```diff
@@ -27,51 +29,57 @@ You can use the prebuilt binaries at the [releases page](https://github.com/Syma
-
-Notice: we have a current bug where the directory locations are not written correctly. For systems using this is equivalent of running keymasted from the cmd/keymasterd directory. For CentOS systems using the rpm the values should be   `data_directory: /var/lib/keymaster ` and `shared_data_directory: /usr/share/keymasterd/`.
-
-###### User password backends
-For password backend keymaster currently supports LDAP backends and apache
-password files. For LDAP the `bind_pattern` is a printf string where `%s` is
-the place where the username will be substituted. For example for an 389ds/openldap
-string might be: `"uid=%s,ou=People,dc=example,dc=com`.
-
-###### U2F token backend
-For u2f/profile backend keymaster supports SQLite and PostgreSQL. The
-`storage_url` field contains the connection information for the database.
-If no `storage_url` is defined keymaster will use an SQLite database located
-in the configured data directory for keymaster.
-
-En example of a postgresql url is:
-`postgresql://dbusername:dbpassword.example.com/keymasterdbname`
-
-###### Unlocking keymasted permanently
-By default the generated config will create an encrypted private key for the keymaster CA. For development, or if your trust model permits it, you can decrypt the private Key and leave it on the filesystem. The encrypted private key is an armored PGP file, so decryption is just `gpg -ad $Filename`. Once decrypted set the `ssh_ca_filename` field in the server config point to this new file.
+The `keymasterd` service runs the following services:
+* **Service Web Interface (default port 443)**: Access to the web interface running on port 443 (default) can be granted via LDAP or apache username/password files. For password backend Keymaster supports LDAP backends and apache password files.
+* **Admin Management Interface (default port 6920)**: The service exposed on port 6920 allows administrators or log collection systems to collect logs generated by the `keymasterd` service.
+
+To run `keymasterd` you will need to generate a config file. `keymasterd` facilitates this through the command-line arguments `-generateConfig` and `-alsoLogToStderr`. Running the `keymasterd` binary with these arguments will generate the following:
+* A configuration file. By default `keymasterd` will write this file to `/etc/keymaster/config.yml`.
+* The Keymaster CA key pair. The encrypted private key  (`masterkey.asc`) is an armored PGP file. For development (or if your trust model permits it) you can decrypt the private-key and write it to the filesystem. To decrypt the key run `gpg -ad $Filename`. Once decrypted set the `ssh_ca_filename` field in the `keymasterd` config file to the path of the decrypted master key.
+* Server keys (for Testing Purposes only): the `server.pem` and `server.key` files are generated to ????
```

Comment:
Testing self signed keys for localhost. This are needed as keymaster requires TLS